README.md

```diff
@@ -1,30 +1,145 @@
 # rav1d
 
-**rav1d** is an **AV1** cross-platform **d**ecoder, open-source, and focused on speed and correctness. It is a Rust port of [dav1d](https://code.videolan.org/videolan/dav1d).
+**rav1d** is an AV1 cross-platform decoder, open-source, and focused on speed
+and correctness. It is a Rust port of
+[dav1d](https://code.videolan.org/videolan/dav1d).
 
-rav1d is currently experimental. Core functionality has been transpiled using [c2rust](https://github.com/immunant/c2rust), but not everything has been ported yet and the transpiled code needs to be cleaned up from unsafe transpiled Rust to safe, idiomatic Rust.
+# Building
+
+rav1d is written in Rust and uses the standard Rust toolchain to build. The Rust
+toolchain can be installed by going to https://rustup.rs. The rav1d library
+builds on stable Rust for x86, x86_64, and aarch64, but currently requires a
+nightly compiler for armv7 and riscv. The project is configured to use a nightly
```

Comment:
Should we mention why?  Since it's a minor issue.

Also, I'm not sure if it's important to be pedantic here, but if we're using `target_arch` names here, then it's just `arm`, not `armv7`.  If we're using `--target` names, then it is `armv7`, but it's `i686` instead of `x86`.  For whichever we use, I think it's also useful to say if we mean `#[cfg(target_arch = "")]`s or `--target`s.

Also, I think we should put these in `` `backticks` ``, in order to contrast with non-specific usages like x86 below, where we really mean `x86` or `x86_64`.

Reply:
Actually I think the `target_arch` names are better, since they're the ones we're actually detecting.  For example, I think `riscv32*` targets should also work since we support `any(target_arch = "riscv32", target_arch = "riscv64")`.

Reply:
Changed to use the `target_arch` names, but I'm leaving riscv32 out because we don't yet check it in CI, so it's not yet a fully supported platform.